packages/verticaltabs/src/react/__specs__/index.spec.tsx

```diff
@@ -16,6 +18,90 @@ describe('VerticalTabs', () => {
     expect(ref).not.toBeNull()
   })
 
+  it('composes className (VerticalTabs)', () => {
+    const { container } = render(<VerticalTabs className="compose-classname" />)
+
+    expect(container.firstChild).toHaveClass(
+      'psds-verticaltabs compose-classname'
+    )
+  })
+
+  it('composes className (VerticalTabs.Divider)', () => {
+    const { container } = render(
+      <VerticalTabs.Divider className="compose-classname" />
+    )
+
+    expect(container.firstChild).toHaveClass(
+      'psds-verticaltabs__divider psds-theme--dark compose-classname'
+    )
+  })
+
+  it('composes className (CollapsibleGroup)', () => {
+    const { container } = render(
+      <CollapsibleGroup className="compose-classname" />
+    )
+
+    expect(container.firstChild).toHaveClass('compose-classname')
```

Comment:
tip for here or the future: 

```
toHaveClass(
  'psds-verticaltabs__divider compose-classname'
)
```

is more valuable than:

```
toHaveClass(
  'compose-classname'
)
```

because it tests _composition_ in addition to props being properly attached to elements.

Reply:
unless of course there is no className on the root node as is the case here. I can rename test to ` properly takes className`